### PR TITLE
#7509 Resolved: ML 8.14 : Single Shot Detector (V)

### LINKED ISSUE #7509: ML 8.14 : Single Shot Detector (V)
### 📌  Issues for Week 8
Welcome to 'ML' Team, good to see you here

:arrow_forward: This issue will helps readers in acquiring all the knowledge that one needs to know about ----
### **_Single Shot Detector_**

:red_circle: To get assigned to this issue, add your **Batch Numbers** mentioned in the spreadsheet of "Machine Learning", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

✔️ Domain : **Machine Learning**

:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title should be the same. Include issue number along with it.

- Changes should be made inside the `Machine Learning/Machine_Learning/ML_for_Sequential_Data_&_Visual Data' Branch.

- Follow Contributing Guidelines & Code of Conduct before start Contributing.

- This issue is only for 'GWOC' contributors of 'Machine Learning' domain.

************************************************************

:white_check_mark: **To be Mentioned while taking the issue :**
- Full name
- Batch Number
- GitHub Profile Link
- Which type of Contribution you want to make : 
    - [ ] Video

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### Domain

Machine Learning

### Type of Contribution

Video

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by VidhiBhatt01**
<hr>

## Description 📜

Added video for Single-shot detector.
Fixes #7509

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [X] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Docu#mentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [X] Machine Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor.

<hr>

### CODE REVIEW

File: Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V) [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+🔴 **Type of Content** : Video
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : ML for Seqential Data & Visual Data
+
+*********************************************************************
+
+### Click here for the audio file,
```
Comment (abhisheks008) [inline Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V):11]: This will be video file

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
The file is not having any .md extension, that's why it's not visible to us

File: Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V) (hunk 1 above)
Comment (VidhiBhatt01) [reply Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V):11]: What do I need to do to fix it? @abhisheks008

**Comment by VidhiBhatt01** [comment]
@abhisheks008 is it visible now?

File: Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V).md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+🔴 **Type of Content** : Video
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : ML for Seqential Data & Visual Data
+
+*********************************************************************
+
+### Click here for the audio file,
```
Comment (abhisheks008) [inline Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_14_Single_Shot_Detector_(V).md:11]: Write here, `Click here for the video file`

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Comment by abhisheks008** [comment]
I have requested some changes for some typos, please do the needful @VidhiBhatt01

**Comment by VidhiBhatt01** [comment]
@abhisheks008 Done.

> I have requested some changes for some typos, please do the needful @VidhiBhatt01

**Review by abhisheks008 — APPROVED** [review]

**Pull request merged**